packages/fxa-auth-server/lib/sentry.js

```diff
@@ -0,0 +1,116 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const Hoek = require('@hapi/hoek');
+const Sentry = require('@sentry/node');
+
+const getVersion = require('./version').getVersion;
+
+// Matches uid, session, oauth and other common tokens which we would
+// prefer not to include in Sentry reports.
+const TOKENREGEX = /[a-fA-F0-9]{32,}/gi;
+const FILTERED = '[Filtered]';
+
+/**
+ * Filter a sentry event for PII in addition to the default filters.
+ *
+ * Current replacements:
+ *   - A 32-char hex string that typically is a FxA user-id.
+ *
+ * Data Removed:
+ *   - Request body.
+ *
+ * @param {Sentry.Event} event
+ */
+function filterSentryEvent(event) {
+  if (event.breadcrumbs) {
+    for (const bc of event.breadcrumbs) {
+      if (bc.message) {
+        bc.message = bc.message.replace(TOKENREGEX, FILTERED);
+      }
+    }
+  }
+  if (event.request) {
+    if (event.request.url) {
+      event.request.url = event.request.url.replace(TOKENREGEX, FILTERED);
+    }
+    const headers = event.request.headers;
+    if (headers) {
+      if (headers.origin) {
```

Comment:
These the only headers? Could we go through all headers and replace fields that match?

Reply:
Sure, added that.